CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,4 @@
+# Code of Conduct
+
+In the interest of fostering an open and welcoming environment, OpenRMF uses the [ROS Code of Conduct](https://osrf.github.io/code_of_conduct/).
```

Comment:
`Open-RMF`

Reply:
https://github.com/open-rmf/rmf-web/pull/705/commits/0adbabe725465854a6102e63e76cd1bf9238156d

Good catch, I copied over from `.github` though, opened https://github.com/open-rmf/.github/pull/3 for that too